pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter.go

```diff
@@ -2431,3 +2427,17 @@ func validateKernelBoot(field *k8sfield.Path, kernelBoot *v1.KernelBoot) (causes
 
 	return
 }
+
+func virtioNicRequested(interfaces []v1.Interface) bool {
+	if len(interfaces) == 0 {
```

Comment:
I find this condition here a bit misleading. If there are no interfaces in means virtio is requested? Not necessary.
I would add this condition explicitly to the place that needs it and not inside the `virtioNicRequested` method.